packages/beacon-node/test/unit/chain/forkChoice/forkChoice.test.ts

```diff
@@ -1,21 +1,20 @@
 import {expect} from "chai";
 import {FAR_FUTURE_EPOCH, MAX_EFFECTIVE_BALANCE} from "@lodestar/params";
 import {config} from "@lodestar/config/default";
-import {phase0, Slot, ssz, ValidatorIndex} from "@lodestar/types";
+import {phase0, ssz} from "@lodestar/types";
 import {CheckpointWithHex, ExecutionStatus, ForkChoice} from "@lodestar/fork-choice";
 import {
   computeEpochAtSlot,
-  getTemporaryBlockHeader,
   CachedBeaconStateAllForks,
   getEffectiveBalanceIncrementsZeroed,
-  processSlots,
 } from "@lodestar/state-transition";
 import {toHexString} from "@chainsafe/ssz";
 import {generateSignedBlock} from "../../../utils/block.js";
 import {generateState} from "../../../utils/state.js";
 import {ChainEventEmitter, computeAnchorCheckpoint, initializeForkChoice} from "../../../../src/chain/index.js";
 import {createCachedBeaconStateTest} from "../../../utils/cachedBeaconState.js";
 import {generateValidators} from "../../../utils/validator.js";
+import {createIndexedAttestation, makeChild, runStateTransition} from "../../../utils/forkChoice.js";
```

Comment:
Utilities that are used only in a single test and will never be used else where can be defined in the same file. The benefit is all code is together, and if they are no longer used, the linter will detected it as not used and make you delete it. Is there a reason to move it out beyond organizational?

Reply:
It's not recommended to export functions from test files. Earlier there was a function which was exported from the test file. 

> Test files should have only one purpose, which is testing a specific unit. Using exports could mean the test file is also used to provide and expose utility or library functionalities, instead those should be moved to separate files.

For details https://github.com/lo1tuma/eslint-plugin-mocha/blob/master/docs/rules/no-exports.md

Reply:
Oh sorry, I missed the exported function below. Then please move the createAttestation one (plus dependant) only.